draft-ietf-deleg.md

```diff
@@ -132,7 +132,7 @@ An NS record contains the hostname of the nameserver for the delegated namespace
 
 In the Domain Name System {{!STD13}}, subdomains within the domain name hierarchy are indicated by delegations to servers which are authoritative for their portion of the namespace.  The DNS records that do this, called NS records, contain hostnames of nameservers, which resolve to addresses.  No other information is available to the resolver. It is limited to connect to the authoritative servers over UDP and TCP port 53. This limitation is a barrier for efficient introduction of new DNS technology.
 
-The proposed DELEG record type remedies this problem by providing extensible parameters to indicate capabilities and additional information, such as glue that a resolver may use for the delegated authority. It is authoritative and thus signed in the parent side of the delegation making it possible to validate all delegation parameters (names and glue records) with DNSSEC.
+The proposed DELEG record type remedies this problem by providing extensible parameters to indicate capabilities and additional information, such as address records that a resolver may use for the delegated authority. It is authoritative and thus signed in the parent side of the delegation making it possible to validate all delegation parameters (names and address records) with DNSSEC.
```

Comment:
To be pedantic, either drop “(names and records)” or if we want to put emphasis maybe formulate as
```suggestion
The proposed DELEG record type remedies this problem by providing extensible parameters to indicate capabilities and additional information, such as address records that a resolver may use for the delegated authority. It is authoritative and thus signed in the parent side of the delegation making it possible to validate all delegation parameters (names and address records included) with DNSSEC.
```

Reply:
I argue these IP addresses are not 'address records' at all. They are IP address literals within DELEG RR, but certainly not 'records' which are a separate entity in the DNS.

Reply:
Good point! It should be "names and addresses"

Reply:
Happy to merge. Can you amend the commit?

Reply:
> I argue these IP addresses are not 'address records' at all. 

fair point.

Reply:
> Personally I would just omit it. Down the road we will add unspecified number of other parameters and I don't see why these two types should be called out.

SGTM

Reply:
```suggestion
The proposed DELEG record type remedies this problem by providing extensible parameters to indicate capabilities and additional information, such as address records that a resolver may use for the delegated authority. It is authoritative and thus signed in the parent side of the delegation making it possible to validate all delegation parameters with DNSSEC.
```